idb/postgres/internal/schema/bigint_schema.go

```diff
@@ -0,0 +1,123 @@
+package schema
```

Comment:
Is there a way we can name this file `test` something or otherwise mark the fact that it's not the actual current production schema?

It doesn't seem like we're maintaining old versions of the schema for past updates. What was our process for previous updates regarding schema testing?